controllers/gatekeeper_controller.go

```diff
@@ -51,3 +119,60 @@ func (r *GatekeeperReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&operatorv1alpha1.Gatekeeper{}).
 		Complete(r)
 }
+
+func (r *GatekeeperReconciler) deployGatekeeperResources() error {
+	ctx := context.Background()
+	var err error
+	for _, a := range orderedStaticAssets {
+		assetName := staticAssetsDir + a
+		bytes, err := bindata.Asset(assetName)
+		if err != nil {
+			return err
+		}
+
+		logger := r.Log.WithValues("Gatekeeper resource", string(assetName))
+		manifest := &manifest.Manifest{}
+		err = manifest.UnmarshalJSON(bytes)
+		if err != nil {
+			return err
```

Comment:
In general, having a more descriptive way of the location of the error will help troubleshooting.
Like wrapping this with "unmarshal failed for resource xxx, err"

Reply:
Fixed